api-misc/src/test/java/org/zalando/nakadi/controller/StoragesControllerTest.java

```diff
@@ -65,7 +63,7 @@ public void testListStorages() throws Exception {
                 .thenReturn(storages);
         when(adminService.isAdmin(AuthorizationService.Operation.READ)).thenReturn(true);
         mockMvc.perform(get("/storages")
-                .principal(mockPrincipal("org/zalando/nakadi")))
+                        .principal(mockPrincipal("org/zalando/nakadi")))
```

Comment:
Do we still check something meaningful here, as we removed the security settings mock above?

Reply:
will check

Reply:
there was nothing meaningful there, it was used in many places only because of historical reasons.